src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -32,7 +32,7 @@ public DependencyContextBuilder(SingleProjectInfo mainProjectInfo, ProjectContex
             _projectContext = projectContext;
 
             // This resolver is only used for building file names, so that base path is not required.
-            _versionFolderPathResolver = new VersionFolderPathResolver(path: null);
+            _versionFolderPathResolver = new VersionFolderPathResolver(null);
```

Comment:
I think that just hides problems. Also longer term it's better to use the parameter name to document the code better.